app/views/donations/_donation_form.html.erb

```diff
@@ -26,7 +26,7 @@
     <div class="col-xs-8 col-md-6 col-lg-3">
     <%= f.association :diaper_drive_participant,
       collection: @diaper_drive_participants,
-      label_method: :business_name,
+      label_method: lambda {|p| !p&.business_name&.empty? ? p.business_name : p&.contact_name },
```

Comment:
Not the hugest fan of how complex the logic here looks - I couldn't use a normal `a || b` type of fallback because it's storing empty strings which don't count as nil.

Reply:
Could we make it default to `nil` instead of `""`?

Reply:
On the frontend probably, but we'd still have to deal with existing db values.